tests/test_helpers.py

```diff
@@ -0,0 +1,56 @@
+import os
+import shutil
+from xonsh.lib import subprocess
+import sys
+
+import pytest
+
+from regolith.broker import load_db
+from regolith.main import main
+
+import openpyxl
+
+helper_map = [
+    (["helper", "hello", "--person", "Simon"],"hello Simon\n")
+        #["a_proprev",
```

Comment:
do you still want these tests?

Reply:
That line will delete when the proprev adder is merged